src/components/offer-card/offer-card.jsx

```diff
@@ -45,21 +60,29 @@ const OfferCard = ({className, offers, onCardClick, setActivePinAction, resetAct
                 <b className="place-card__price-value">&euro;{offer.price}</b>
                 <span className="place-card__price-text">&#47;&nbsp;night</span>
               </div>
-              <button
-                className={offer.isFavorite ? `place-card__bookmark-button--active button` : `place-card__bookmark-button button`} type="button">
-                <svg className="place-card__bookmark-icon" width="18" height="19">
-                  <use xlinkHref="#icon-bookmark"></use>
-                </svg>
-                <span className="visually-hidden">To bookmarks</span>
-              </button>
+              <ListBookmarkButton
+                className = {className}
+                offer = {offer}
+                authorizationStatus = {authorizationStatus}
+                changeFavoriteAction = {changeFavoriteAction}
+                refreshOfferList = {refreshOfferList}
+                redirectToRoute = {redirectToRouteAction}
+                currentOfferId = {currentOfferId}
+                refreshNearOffersList = {refreshNearOffersList}
```

Comment:
в описании пропсов пробелы обычно не делают
```suggestion
                className = {className}
                offer = {offer}
                authorizationStatus={authorizationStatus}
                changeFavoriteAction={changeFavoriteAction}
                refreshOfferList={refreshOfferList}
                redirectToRoute={redirectToRouteAction}
                currentOfferId={currentOfferId}
                refreshNearOffersList={refreshNearOffersList}
```